addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2392,63 +2394,37 @@ registry.category("snippet_widgets").add("WeDatetime", WeDatetime);
 
 const DatePickerUserValueWidget = DatetimePickerUserValueWidget.extend({});
 
-const ListUserValueWidget = UserValueWidget.extend({
-    tagName: 'we-list',
-    events: {
-        'click we-button.o_we_select_remove_option': '_onRemoveItemClick',
-        'click we-button.o_we_list_add_optional': '_onAddCustomItemClick',
-        'click we-button.o_we_list_add_existing': '_onAddExistingItemClick',
-        'click we-select.o_we_user_value_widget.o_we_add_list_item': '_onAddItemSelectClick',
-        'click we-button.o_we_checkbox_wrapper': '_onAddItemCheckboxClick',
-        'input table input': '_onListItemBlurInput',
-        'blur table input': '_onListItemBlurInput',
-        'mousedown': '_onWeListMousedown',
-    },
+const ListUserValueWidget = UserValueWidget.extend({});
+class ListUserValue extends UserValue {
+    constructor() {
+        super(...arguments);
+        Object.assign(this._state, {
+            listRecords: [],
+            selected: [],
+        });
```

Comment:
I don't have a strong opinion. It's just that I had a bunch more properties at first, so this was cleaner, and if we ever need to add new state properties this would be cleaner too I think (in the sense that everything would be gathered at the same place).
But it's true that, as is with 2 properties, it doesn't matter much. :man_shrugging: